src/libs/testing-library/utils.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  encode,
+  RootRoute,
+  Route,
+  Router,
+  createMemoryHistory,
+} from '@tanstack/react-router';
+import { RouterRenderParams } from './types';
+import { debugTokens } from '../../../e2e/utils/types';
+
+export const tokens = [
+  {
+    address: debugTokens.ETH,
+    decimals: 18,
+    symbol: 'ETH',
+  },
+  {
+    address: debugTokens.USDC,
+    decimals: 6,
+    symbol: 'USDC',
+  },
+];
+
+export const loadRouter = async ({
```

Comment:
Could you add some comments inside this function to understand the process ?

Reply:
Added jsdoc